core/src/main/java/io/fabric8/maven/core/service/BuildService.java

```diff
@@ -43,6 +43,9 @@
      */
     void postProcess(BuildServiceConfig config);
 
+    default BuildServiceConfig getBuildServiceConfig() {
```

Comment:
This method is never implemented neither invoked, it's cluttering the interface, we should probably remove it.